ColabTool/Shared/Post/story.jsx

```diff
@@ -9,18 +9,24 @@ import { Text } from '@bufferapp/components';
 import Post from './index';
 
```

Comment:
Heya @stevemdixon - this is looking good so far! I think it could be useful to have some extra tests/stories here for retweets with and without comments 😄